docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.1"
+
+services:
+  nextcloud:
+    image: nextcloud/all-in-one
+    restart: unless-stopped
+    container_name: nextcloud-aio
```

Comment:
how does the resulting container name look like?
Could you post the output of docker ps here?
It needs to be nextcloud-aio-mastercontainer otherwise the built-in updater will not work.

Reply:
```
CONTAINER ID   IMAGE                         COMMAND                  CREATED          STATUS          PORTS                                                                                          NAMES
ac3e7526ab75   nextcloud/all-in-one          "start.sh /usr/bin/s…"   5 hours ago      Up 11 seconds   0.0.0.0:8080->8080/tcp, :::8080->8080/tcp, 80/tcp, 0.0.0.0:8443->8443/tcp, :::8443->8443/tcp   nextcloud-aio
```

> It needs to be nextcloud-aio-mastercontainer otherwise the built-in updater will not work.

ah I didn't know that, container_name says how the container will be named.

Reply:
I'll change it to `nextcloud-aio-mastercontainer`